client/src/components/Applications/AppForm/AppForm.scss

```diff
@@ -1,23 +1,55 @@
 #appDetail-wrapper {
+    --space-top: 1.25rem; /* 20px (~17px design * 1.2 design-to-app ratio) */
+    --space-bottom: 2.25rem; /* 36px (~30px design * 1.2 design-to-app ratio) */
+
+    /* As a flex item */
     flex-grow: 1;
+
+    /* As a flex container */
     display: flex;
     flex-direction: row;
-    height: 100%; /* Be consistent with `#appDetail-error`/`.app-placeholder` (which can exist at the same location),
-                     so that `appBrowser-wrapper` height does not change based on sibling content */
-    padding-top: 15px;
-    /* Match bottom space found in UI design at `Core Portal > Applications` */
-    padding-bottom: 30px;
```

Comment:
I think we had this to also pad the bottom of HTML apps - that's gone now. Desired?
This PR (new):
![Screen Shot 2021-05-12 at 3 16 56 PM](https://user-images.githubusercontent.com/20326896/118048880-8d041300-b342-11eb-9438-1ceeb6674205.png)
Current:
<img width="1580" alt="Screen Shot 2021-05-12 at 3 17 00 PM" src="https://user-images.githubusercontent.com/20326896/118048876-8bd2e600-b342-11eb-8c54-66a3b498d966.png">

Reply:
Perhaps lost here: https://github.com/TACC/Core-Portal/pull/350/files#diff-27808d3ea5a84433bfaae58267ae9fcddd1811043dfb2bb8002a2e48897aaad0L88

Reply:
It was an intentional change. The rounded corners should not be there, those styles come from Bootstrap Jumbotron and do not match the design. The margin was difficult to maintain. Now that the components are more finished (and my mind is outside of that box), I may see be able to see a solution.

I'll confirm all the changes here with Design. [Questions asked.](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1620939845081900)

Reply:
👍 Just wanted to make sure it was intentional.

Reply:
I have [Design approval](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1620940014083500).